docs/Venture - Engineering Requirements.md

```
@@ -0,0 +1,268 @@
+
+Request: POST /upload/presign
+
+{  
+"contentType": "image/jpeg",  
+"fileExt": "jpg"  
+}
+
+Response:
+
+{  
+"url": "https://s3.amazonaws.com/....",  
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)